specs/subresourceintegrity/spec.markdown

```diff
@@ -263,9 +263,9 @@ list, and use that metadata to validate the resource (as described below in
 the "[parse metadata][parse]" and "[get the strongest metadata from
 set][get-the-strongest]" algorithms).
 
-When a hash function is determined to be insecure, user agents MUST deprecate
-and eventually remove support for integrity validation using that hash
-function.
+User agents MAY deprecate support (by blocking loads) for integrity validation using 
```

Comment:
I'm ok with this, but we should probably mention this on the list first since it is a departure from our otherwise "fail open" policy.